docs/sources/shared/visualizations/disconnect-values.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Disconnect values
+---
+
+### Disconnect values
+
+Choose whether to set a threshold above which values in the data should be disconnected.
+
+{{< figure src="/media/docs/grafana/screenshot-grafana-10-1-disconnect-values.png" max-width="750px" caption="Disconnect values toggle" >}}
```

Comment:
Tiny nit, i don't think we need the caption here. Seems like overkill. 

![image](https://github.com/grafana/grafana/assets/76437239/99a55760-1a70-439c-8fbc-86b45cdd9e66)

```suggestion
{{< figure src="/media/docs/grafana/screenshot-grafana-10-1-disconnect-values.png" max-width="750px" >}}
```

Reply:
Thanks for pointing this out :) I was under impression that we should always include captions for improved a11y 🤷‍♂️ (screen readers etc.) - I agree though especially for this image it is pretty redundant so I'm ok with taking it out 👍